src/utils/lightBlock.ts

```diff
@@ -0,0 +1,50 @@
+import { GetBlockResponse, FollowChainStreamResponse } from "@ironfish/sdk";
```

Comment:
Only renamed this file.

Reply:
nit: if you use `git mv` this diff shouldn't happen. Also I think if you move via VSCode UI I would think this diff shouldn't happen.